src/patternfly/components/Icon/icon.scss

```diff
@@ -0,0 +1,134 @@
+.pf-c-icon {
+  // Example block variable
+  --pf-c-icon--Width: var(--pf-global--icon--FontSize--md);
+  --pf-c-icon--Height: var(--pf-global--icon--FontSize--md);
+
+  // Size modifiers
+  --pf-c-icon--m-sm--Width: var(--pf-global--icon--FontSize--sm);
+  --pf-c-icon--m-md--Width: var(--pf-global--icon--FontSize--md);
+  --pf-c-icon--m-lg--Width: var(--pf-global--icon--FontSize--lg);
+  --pf-c-icon--m-xl--Width: var(--pf-global--icon--FontSize--xl);
+  --pf-c-icon--m-sm--Height: var(--pf-global--icon--FontSize--sm);
+  --pf-c-icon--m-md--Height: var(--pf-global--icon--FontSize--md);
+  --pf-c-icon--m-lg--Height: var(--pf-global--icon--FontSize--lg);
+  --pf-c-icon--m-xl--Height: var(--pf-global--icon--FontSize--xl);
+
+  // Content variables
+  --pf-c-icon__content--Color: var(--pf-global--icon--Color--dark);
+
+  // Content status TODO: do we want to do this?
+  --pf-c-icon__content-m-danger--Color: var(--pf-global--danger-color--100);
+  --pf-c-icon__content-m-warning--Color: var(--pf-global--warning-color--100);
+  --pf-c-icon__content-m-success--Color: var(--pf-global--success-color--100);
+  --pf-c-icon__content-m-info--Color: var(--pf-global--info-color--100);
+  --pf-c-icon__content-m-default--Color: var(--pf-global--default-color--100);
+  --pf-c-icon__content--FontSize: var(--pf-global--icon--FontSize--md);
+  --pf-c-icon__content-svg--VerticalAlign: -.125em;
```

Comment:
I'm not sure why to add the extra hyphen here; it's a sub element but instead of chaining a__b__c we usually do something like a__b-c, right?

Reply:
I don't think so, do you have an example?

Since `__el-name` is how we denote an element with a space in it's name (like `.pf-c-icon__content-svg`), that could create a conflict. When we're styling a child of a thing, we use `--[name]`, like `--svg` here since we're targeting a generic svg, or `--child` as any generic child (that's a common one), or `--c-[component]` if we're targeting a component as a child. That shouldn't create conflicts since we use `--` with `--m-[modifier]`, `--[hover/focus/etc]`, `--[before/after]`, `--[breakpoint]`, which all have specific names that aren't likely to overlap.

Reply:
Our guidelines don't cover this case - the rule is given as `--pf-c-block[__element][--modifier][--state][--breakpoint][--pseudo-element]--PropertyCamelCase` but it was done in Login https://github.com/patternfly/patternfly/blob/f686022670816901155c27b22d39006d1f599fb2/src/patternfly/components/Login/login.scss#L256

Reply:
Do you mind opening an issue to add that to the docs? Here are some examples:

```
$ grep -Er -- '^\s+--pf-' src/patternfly/components/ | cut -d':' -f2 | grep -Ei -- '--(child|item|link|c-)' | awk '{$1=$1};1' | sort
--pf-c-about-modal-box__close--c-button--BackgroundColor
--pf-c-about-modal-box__close--c-button--BackgroundColor
--pf-c-about-modal-box__close--c-button--BorderRadius
--pf-c-about-modal-box__close--c-button--Color
--pf-c-about-modal-box__close--c-button--FontSize
--pf-c-about-modal-box__close--c-button--Height
--pf-c-about-modal-box__close--c-button--Width
--pf-c-about-modal-box__close--c-button--hover--BackgroundColor
--pf-c-action-list--child--spacer
--pf-c-action-list--child--spacer
--pf-c-action-list--child--spacer-base
--pf-c-app-launcher--c-divider--MarginBottom
--pf-c-app-launcher--c-divider--MarginBottom
--pf-c-app-launcher--c-divider--MarginTop
--pf-c-back-to-top--c-button--BorderRadius
--pf-c-back-to-top--c-button--BoxShadow
--pf-c-back-to-top--c-button--FontSize
--pf-c-back-to-top--c-button--PaddingBottom
--pf-c-back-to-top--c-button--PaddingLeft
--pf-c-back-to-top--c-button--PaddingRight
--pf-c-back-to-top--c-button--PaddingTop
--pf-c-banner--link--Color
--pf-c-banner--link--Color
--pf-c-banner--link--Color
--pf-c-banner--link--Color
--pf-c-banner--link--TextDecoration
--pf-c-banner--link--TextDecoration
--pf-c-banner--link--TextDecoration
--pf-c-banner--link--disabled--Color
--pf-c-banner--link--disabled--TextDecoration
--pf-c-banner--link--hover--Color
--pf-c-banner--link--hover--FontWeight
--pf-c-breadcrumb__dropdown--c-dropdown__toggle--LineHeight
--pf-c-card--c-divider--child--PaddingTop
--pf-c-card--c-divider--child--PaddingTop
--pf-c-card--c-divider--child--PaddingTop
--pf-c-card--child--PaddingBottom
--pf-c-card--child--PaddingBottom
--pf-c-card--child--PaddingBottom
--pf-c-card--child--PaddingLeft
--pf-c-card--child--PaddingLeft
--pf-c-card--child--PaddingLeft
--pf-c-card--child--PaddingRight
--pf-c-card--child--PaddingRight
--pf-c-card--child--PaddingRight
--pf-c-card--m-compact--c-divider--child--PaddingTop
--pf-c-card--m-compact--child--PaddingBottom
--pf-c-card--m-compact--child--PaddingLeft
--pf-c-card--m-compact--child--PaddingRight
--pf-c-card--m-display-lg--c-divider--child--PaddingTop
--pf-c-card--m-display-lg--child--PaddingBottom
--pf-c-card--m-display-lg--child--PaddingLeft
--pf-c-card--m-display-lg--child--PaddingRight
--pf-c-card__actions--child--MarginLeft
--pf-c-code-editor__controls--c-button--m-control--Color
--pf-c-code-editor__controls--c-button--m-control--Color
--pf-c-code-editor__controls--c-button--m-control--Color
--pf-c-code-editor__controls--c-button--m-control--disabled--after--BorderBottomColor
--pf-c-code-editor__controls--c-button--m-control--focus--Color
--pf-c-code-editor__controls--c-button--m-control--hover--Color
--pf-c-data-list__item--m-selectable--hover--item--BorderTopColor
--pf-c-data-list__item--m-selectable--hover--item--BorderTopWidth
--pf-c-date-picker__input--c-form-control--Width
--pf-c-date-picker__input--c-form-control--width-base
--pf-c-date-picker__input--c-form-control--width-chars
--pf-c-drawer--child--PaddingBottom
--pf-c-drawer--child--PaddingBottom
--pf-c-drawer--child--PaddingLeft
--pf-c-drawer--child--PaddingLeft
--pf-c-drawer--child--PaddingRight
--pf-c-drawer--child--PaddingRight
--pf-c-drawer--child--PaddingTop
--pf-c-drawer--child--PaddingTop
--pf-c-drawer--child--m-padding--PaddingBottom
--pf-c-drawer--child--m-padding--PaddingBottom
--pf-c-drawer--child--m-padding--PaddingLeft
--pf-c-drawer--child--m-padding--PaddingLeft
--pf-c-drawer--child--m-padding--PaddingRight
--pf-c-drawer--child--m-padding--PaddingRight
--pf-c-drawer--child--m-padding--PaddingTop
--pf-c-drawer--child--m-padding--PaddingTop
--pf-c-drawer--child--m-padding--md--PaddingBottom
--pf-c-drawer--child--m-padding--md--PaddingLeft
--pf-c-drawer--child--m-padding--md--PaddingRight
--pf-c-drawer--child--m-padding--md--PaddingTop
--pf-c-drawer--child--md--PaddingBottom
--pf-c-drawer--child--md--PaddingLeft
--pf-c-drawer--child--md--PaddingRight
--pf-c-drawer--child--md--PaddingTop
--pf-c-drawer__content--child--PaddingBottom
--pf-c-drawer__content--child--PaddingLeft
--pf-c-drawer__content--child--PaddingRight
--pf-c-drawer__content--child--PaddingTop
--pf-c-dropdown--c-divider--MarginBottom
--pf-c-dropdown--c-divider--MarginBottom
--pf-c-dropdown--c-divider--MarginTop
--pf-c-dropdown--c-menu--Top
--pf-c-dropdown--c-menu--Top
--pf-c-dropdown--c-menu--ZIndex
--pf-c-dropdown--m-expanded__toggle--m-secondary--m-split-button--child--before--BorderWidth
--pf-c-dropdown--m-top--c-menu--Top
--pf-c-dropdown--m-top--c-menu--TranslateY
--pf-c-dropdown__toggle--c-badge__toggle-icon--MarginLeft
--pf-c-dropdown__toggle--c-badge__toggle-icon--MarginRight
--pf-c-dropdown__toggle--m-plain--child--LineHeight
--pf-c-dropdown__toggle--m-secondary--m-split-button--child--before--BorderColor
--pf-c-dropdown__toggle--m-secondary--m-split-button--child--before--BorderColor--base
--pf-c-dropdown__toggle--m-secondary--m-split-button--child--before--BorderWidth
--pf-c-dropdown__toggle--m-secondary--m-split-button--child--before--BorderWidth
--pf-c-dropdown__toggle--m-secondary--m-split-button--child--before--BorderWidth
--pf-c-dropdown__toggle--m-secondary--m-split-button--child--before--BorderWidth
--pf-c-dropdown__toggle--m-secondary--m-split-button--child--before--BorderWidth
--pf-c-dropdown__toggle--m-secondary--m-split-button--child--before--BorderWidth--base
--pf-c-dropdown__toggle--m-secondary--m-split-button--child--before--active--BorderWidth
--pf-c-dropdown__toggle--m-secondary--m-split-button--child--before--focus--BorderWidth
--pf-c-dropdown__toggle--m-secondary--m-split-button--child--before--hover--BorderWidth
--pf-c-dropdown__toggle--m-split-button--child--BackgroundColor
--pf-c-dropdown__toggle--m-split-button--child--BorderRadius
--pf-c-dropdown__toggle--m-split-button--child--PaddingBottom
--pf-c-dropdown__toggle--m-split-button--child--PaddingLeft
--pf-c-dropdown__toggle--m-split-button--child--PaddingLeft
--pf-c-dropdown__toggle--m-split-button--child--PaddingLeft
--pf-c-dropdown__toggle--m-split-button--child--PaddingRight
--pf-c-dropdown__toggle--m-split-button--child--PaddingRight
--pf-c-dropdown__toggle--m-split-button--child--PaddingRight
--pf-c-dropdown__toggle--m-split-button--child--PaddingTop
--pf-c-dropdown__toggle--m-split-button--m-action--child--PaddingLeft
--pf-c-dropdown__toggle--m-split-button--m-action--child--PaddingRight
--pf-c-dropdown__toggle--m-split-button--m-primary--child--BackgroundColor
--pf-c-dropdown__toggle--m-split-button--m-primary--child--BackgroundColor
--pf-c-dropdown__toggle--m-split-button--m-primary--child--BackgroundColor
--pf-c-dropdown__toggle--m-split-button--m-primary--child--BackgroundColor
--pf-c-dropdown__toggle--m-split-button--m-primary--child--BackgroundColor
--pf-c-dropdown__toggle--m-split-button--m-primary--child--BorderRadius
--pf-c-dropdown__toggle--m-split-button--m-primary--child--active--BackgroundColor
--pf-c-dropdown__toggle--m-split-button--m-primary--child--focus--BackgroundColor
--pf-c-dropdown__toggle--m-split-button--m-primary--child--hover--BackgroundColor
--pf-c-dropdown__toggle--m-split-button--m-primary--child--m-expanded--BackgroundColor
--pf-c-dropdown__toggle--m-split-button--m-primary--m-action--child--BorderLeftColor
--pf-c-dropdown__toggle--m-split-button--m-primary--m-action--child--BorderLeftWidth
--pf-c-dropdown__toggle--m-split-button--m-primary--m-action--child--BorderLeftWidth
--pf-c-dual-list-selector__draggable--item--PaddingLeft
--pf-c-dual-list-selector__item--c-badge--m-read--BackgroundColor
--pf-c-empty-state--c-button__secondary--MarginTop
--pf-c-empty-state--m-xl--c-button__secondary--MarginTop
--pf-c-empty-state__content--c-title--m-lg--FontSize
--pf-c-empty-state__secondary--child--MarginBottom
--pf-c-empty-state__secondary--child--MarginLeft
--pf-c-empty-state__secondary--child--MarginRight
--pf-c-form-control--placeholder--child--Color
--pf-c-form__actions--child--MarginBottom
--pf-c-form__actions--child--MarginLeft
--pf-c-form__actions--child--MarginRight
--pf-c-form__actions--child--MarginTop
--pf-c-form__group-control--m-inline--child--MarginRight
--pf-c-form__group-control--m-inline--child--MarginRight
--pf-c-hint__actions--c-dropdown--MarginTop
--pf-c-hint__footer--child--MarginRight
--pf-c-inline-edit__action--c-button--m-valid--m-plain--Color
--pf-c-inline-edit__action--c-button--m-valid--m-plain--hover--Color
--pf-c-inline-edit__action--m-icon-group--item--MarginRight
--pf-c-inline-edit__group--item--MarginRight
--pf-c-inline-edit__group--item--MarginRight
--pf-c-inline-edit__group--item--MarginRight
--pf-c-inline-edit__group--item--MarginRight
--pf-c-input-group--c-form-control--MarginRight
--pf-c-input-group--c-form-control--invalid--ZIndex
--pf-c-input-group--child--ZIndex
--pf-c-label--m-blue__content--link--focus--before--BorderColor
--pf-c-label--m-blue__content--link--hover--before--BorderColor
--pf-c-label--m-cyan__content--link--focus--before--BorderColor
--pf-c-label--m-cyan__content--link--hover--before--BorderColor
--pf-c-label--m-gold__content--link--focus--before--BorderColor
--pf-c-label--m-gold__content--link--hover--before--BorderColor
--pf-c-label--m-green__content--link--focus--before--BorderColor
--pf-c-label--m-green__content--link--hover--before--BorderColor
--pf-c-label--m-orange__content--link--focus--before--BorderColor
--pf-c-label--m-orange__content--link--hover--before--BorderColor
--pf-c-label--m-outline--m-blue__content--link--focus--before--BorderColor
--pf-c-label--m-outline--m-blue__content--link--focus--before--BorderColor
--pf-c-label--m-outline--m-blue__content--link--hover--before--BorderColor
--pf-c-label--m-outline--m-blue__content--link--hover--before--BorderColor
--pf-c-label--m-outline--m-cyan__content--link--focus--before--BorderColor
--pf-c-label--m-outline--m-cyan__content--link--focus--before--BorderColor
--pf-c-label--m-outline--m-cyan__content--link--hover--before--BorderColor
--pf-c-label--m-outline--m-cyan__content--link--hover--before--BorderColor
--pf-c-label--m-outline--m-gold__content--link--focus--before--BorderColor
--pf-c-label--m-outline--m-gold__content--link--hover--before--BorderColor
--pf-c-label--m-outline--m-green__content--link--focus--before--BorderColor
--pf-c-label--m-outline--m-green__content--link--focus--before--BorderColor
--pf-c-label--m-outline--m-green__content--link--hover--before--BorderColor
--pf-c-label--m-outline--m-green__content--link--hover--before--BorderColor
--pf-c-label--m-outline--m-orange__content--link--focus--before--BorderColor
--pf-c-label--m-outline--m-orange__content--link--focus--before--BorderColor
--pf-c-label--m-outline--m-orange__content--link--hover--before--BorderColor
--pf-c-label--m-outline--m-orange__content--link--hover--before--BorderColor
--pf-c-label--m-outline--m-purple__content--link--focus--before--BorderColor
--pf-c-label--m-outline--m-purple__content--link--focus--before--BorderColor
--pf-c-label--m-outline--m-purple__content--link--hover--before--BorderColor
--pf-c-label--m-outline--m-purple__content--link--hover--before--BorderColor
--pf-c-label--m-outline--m-red__content--link--focus--before--BorderColor
--pf-c-label--m-outline--m-red__content--link--focus--before--BorderColor
--pf-c-label--m-outline--m-red__content--link--hover--before--BorderColor
--pf-c-label--m-outline--m-red__content--link--hover--before--BorderColor
--pf-c-label--m-outline__content--link--focus--before--BorderColor
--pf-c-label--m-outline__content--link--focus--before--BorderColor
--pf-c-label--m-outline__content--link--focus--before--BorderColor
--pf-c-label--m-outline__content--link--focus--before--BorderColor
--pf-c-label--m-outline__content--link--focus--before--BorderColor
--pf-c-label--m-outline__content--link--focus--before--BorderColor
--pf-c-label--m-outline__content--link--focus--before--BorderColor
--pf-c-label--m-outline__content--link--focus--before--BorderColor
--pf-c-label--m-outline__content--link--focus--before--BorderWidth
--pf-c-label--m-outline__content--link--focus--before--BorderWidth
--pf-c-label--m-outline__content--link--hover--before--BorderColor
--pf-c-label--m-outline__content--link--hover--before--BorderColor
--pf-c-label--m-outline__content--link--hover--before--BorderColor
--pf-c-label--m-outline__content--link--hover--before--BorderColor
--pf-c-label--m-outline__content--link--hover--before--BorderColor
--pf-c-label--m-outline__content--link--hover--before--BorderColor
--pf-c-label--m-outline__content--link--hover--before--BorderColor
--pf-c-label--m-outline__content--link--hover--before--BorderColor
--pf-c-label--m-outline__content--link--hover--before--BorderWidth
--pf-c-label--m-outline__content--link--hover--before--BorderWidth
--pf-c-label--m-overflow__content--link--focus--before--BorderColor
--pf-c-label--m-overflow__content--link--focus--before--BorderWidth
--pf-c-label--m-overflow__content--link--hover--before--BorderColor
--pf-c-label--m-overflow__content--link--hover--before--BorderWidth
--pf-c-label--m-purple__content--link--focus--before--BorderColor
--pf-c-label--m-purple__content--link--focus--before--BorderColor
--pf-c-label--m-purple__content--link--hover--before--BorderColor
--pf-c-label--m-purple__content--link--hover--before--BorderColor
--pf-c-label--m-red__content--link--focus--before--BorderColor
--pf-c-label--m-red__content--link--hover--before--BorderColor
--pf-c-label-group--m-vertical__close--c-button--PaddingLeft
--pf-c-label-group--m-vertical__close--c-button--PaddingRight
--pf-c-label__content--link--focus--before--BorderColor
--pf-c-label__content--link--focus--before--BorderColor
--pf-c-label__content--link--focus--before--BorderColor
--pf-c-label__content--link--focus--before--BorderColor
--pf-c-label__content--link--focus--before--BorderColor
--pf-c-label__content--link--focus--before--BorderColor
--pf-c-label__content--link--focus--before--BorderColor
--pf-c-label__content--link--focus--before--BorderColor
--pf-c-label__content--link--focus--before--BorderColor
--pf-c-label__content--link--focus--before--BorderColor
--pf-c-label__content--link--focus--before--BorderWidth
--pf-c-label__content--link--focus--before--BorderWidth
--pf-c-label__content--link--focus--before--BorderWidth
--pf-c-label__content--link--focus--before--BorderWidth
--pf-c-label__content--link--hover--before--BorderColor
--pf-c-label__content--link--hover--before--BorderColor
--pf-c-label__content--link--hover--before--BorderColor
--pf-c-label__content--link--hover--before--BorderColor
--pf-c-label__content--link--hover--before--BorderColor
--pf-c-label__content--link--hover--before--BorderColor
--pf-c-label__content--link--hover--before--BorderColor
--pf-c-label__content--link--hover--before--BorderColor
--pf-c-label__content--link--hover--before--BorderColor
--pf-c-label__content--link--hover--before--BorderColor
--pf-c-label__content--link--hover--before--BorderWidth
--pf-c-label__content--link--hover--before--BorderWidth
--pf-c-label__content--link--hover--before--BorderWidth
--pf-c-label__content--link--hover--before--BorderWidth
--pf-c-log-viewer--c-toolbar--PaddingBottom
--pf-c-log-viewer--c-toolbar--PaddingTop
--pf-c-log-viewer--c-toolbar__content--PaddingLeft
--pf-c-log-viewer--c-toolbar__content--PaddingRight
--pf-c-log-viewer--c-toolbar__group--m-toggle-group--m-show--spacer
--pf-c-log-viewer--c-toolbar__group--m-toggle-group--spacer
--pf-c-login__footer--c-list--PaddingTop
--pf-c-login__footer--c-list--PaddingTop
--pf-c-login__footer--c-list--xl--PaddingTop
--pf-c-login__header--c-brand--MarginBottom
--pf-c-login__header--c-brand--MarginBottom
--pf-c-login__header--c-brand--xl--MarginBottom
--pf-c-login__main-footer--c-title--MarginBottom
--pf-c-masthead--c-context-selector--Width
--pf-c-masthead--c-context-selector--m-full-height__toggle--BorderBottomColor
--pf-c-masthead--c-context-selector--m-full-height__toggle--BorderTopColor
--pf-c-masthead--c-context-selector__toggle--BorderLeftColor
--pf-c-masthead--c-context-selector__toggle--BorderRightColor
--pf-c-masthead--c-context-selector__toggle--BorderTopColor
--pf-c-masthead--c-dropdown--m-full-height__toggle--before--BorderBottomColor
--pf-c-masthead--c-dropdown--m-full-height__toggle--before--BorderTopColor
--pf-c-masthead--c-dropdown__toggle--before--BorderLeftColor
--pf-c-masthead--c-dropdown__toggle--before--BorderRightColor
--pf-c-masthead--c-dropdown__toggle--before--BorderTopColor
--pf-c-masthead--c-menu-toggle--before--BorderLeftColor
--pf-c-masthead--c-menu-toggle--before--BorderRightColor
--pf-c-masthead--c-menu-toggle--before--BorderTopColor
--pf-c-masthead--c-menu-toggle--m-full-height--before--BorderBottomColor
--pf-c-masthead--c-menu-toggle--m-full-height--before--BorderTopColor
--pf-c-masthead--c-toolbar__content--PaddingLeft
--pf-c-masthead--c-toolbar__content--PaddingRight
--pf-c-masthead--c-toolbar__expandable-content--BorderTopColor
--pf-c-masthead--c-toolbar__expandable-content--BorderTopWidth
--pf-c-masthead--c-toolbar__expandable-content--PaddingBottom
--pf-c-masthead--c-toolbar__expandable-content--PaddingLeft
--pf-c-masthead--c-toolbar__expandable-content--PaddingRight
--pf-c-masthead--c-toolbar__expandable-content--PaddingTop
--pf-c-masthead--c-toolbar__item--Display
--pf-c-masthead--item-border-color--base
--pf-c-masthead--m-display-inline__content--c-nav--m-horizontal--MarginLeft
--pf-c-masthead--m-display-inline__content--c-nav--m-horizontal--MarginRight
--pf-c-masthead--m-display-stack__content--c-nav--m-horizontal--MarginLeft
--pf-c-masthead--m-display-stack__content--c-nav--m-horizontal--MarginRight
--pf-c-masthead__content--c-nav--m-horizontal--MarginLeft
--pf-c-masthead__content--c-nav--m-horizontal--MarginLeft
--pf-c-masthead__content--c-nav--m-horizontal--MarginLeft
--pf-c-masthead__content--c-nav--m-horizontal--MarginRight
--pf-c-masthead__content--c-nav--m-horizontal--MarginRight
--pf-c-masthead__toggle--c-button--FontSize
--pf-c-menu--c-divider--MarginBottom
--pf-c-menu--c-divider--MarginTop
--pf-c-menu--m-drilldown--c-menu--Top
--pf-c-menu--m-drilldown--c-menu--Transition
--pf-c-menu--m-drilldown--c-menu--TransitionDuration--transform
--pf-c-menu--m-drilled-in--c-menu__list-item--m-current-path--c-menu--ZIndex
--pf-c-menu--m-nav--c-menu--Left
--pf-c-menu--m-nav--c-menu--Left
--pf-c-menu--m-nav--c-menu--Top
--pf-c-menu--m-nav--c-menu--Top
--pf-c-menu--m-nav--c-menu--Top
--pf-c-menu--m-nav--c-menu--left-offset
--pf-c-menu--m-nav--c-menu--m-left--Right
--pf-c-menu--m-nav--c-menu--m-left--right-offset
--pf-c-menu--m-nav--c-menu--m-top--Bottom
--pf-c-menu--m-nav--c-menu--m-top--bottom-offset
--pf-c-menu--m-nav__list-item--first-child--c-menu--Top
--pf-c-menu-toggle--m-split-button--child--BackgroundColor
--pf-c-menu-toggle--m-split-button--child--BackgroundColor
--pf-c-menu-toggle--m-split-button--child--Color
--pf-c-menu-toggle--m-split-button--child--disabled--BackgroundColor
--pf-c-menu-toggle--m-split-button--child--disabled--Color
--pf-c-menu-toggle--m-split-button--m-action--child--BorderLeftColor
--pf-c-menu-toggle--m-split-button--m-action--child--BorderLeftColor
--pf-c-menu-toggle--m-split-button--m-action--child--BorderLeftColor
--pf-c-menu-toggle--m-split-button--m-action--child--BorderLeftColor
--pf-c-menu-toggle--m-split-button--m-action--child--BorderLeftColor
--pf-c-menu-toggle--m-split-button--m-action--child--BorderLeftWidth
--pf-c-menu-toggle--m-split-button--m-action--child--BorderRadius
--pf-c-menu-toggle--m-split-button--m-action--child--active--after--BorderBottomColor
--pf-c-menu-toggle--m-split-button--m-action--child--active--after--BorderBottomWidth
--pf-c-menu-toggle--m-split-button--m-action--child--after--BorderBottomColor
--pf-c-menu-toggle--m-split-button--m-action--child--after--BorderBottomColor
--pf-c-menu-toggle--m-split-button--m-action--child--after--BorderBottomColor
--pf-c-menu-toggle--m-split-button--m-action--child--after--BorderBottomColor
--pf-c-menu-toggle--m-split-button--m-action--child--after--BorderBottomWidth
--pf-c-menu-toggle--m-split-button--m-action--child--after--BorderBottomWidth
--pf-c-menu-toggle--m-split-button--m-action--child--after--BorderBottomWidth
--pf-c-menu-toggle--m-split-button--m-action--child--after--BorderBottomWidth
--pf-c-menu-toggle--m-split-button--m-action--child--after--BorderBottomWidth
--pf-c-menu-toggle--m-split-button--m-action--child--after--BorderBottomWidth
--pf-c-menu-toggle--m-split-button--m-action--child--after--Left
--pf-c-menu-toggle--m-split-button--m-action--child--after--Left
--pf-c-menu-toggle--m-split-button--m-action--child--focus--after--BorderBottomColor
--pf-c-menu-toggle--m-split-button--m-action--child--focus--after--BorderBottomWidth
--pf-c-menu-toggle--m-split-button--m-action--child--hover--after--BorderBottomColor
--pf-c-menu-toggle--m-split-button--m-action--child--hover--after--BorderBottomWidth
--pf-c-menu-toggle--m-split-button--m-action--m-primary--child--BackgroundColor
--pf-c-menu-toggle--m-split-button--m-action--m-primary--child--BackgroundColor
--pf-c-menu-toggle--m-split-button--m-action--m-primary--child--BackgroundColor
--pf-c-menu-toggle--m-split-button--m-action--m-primary--child--BackgroundColor
--pf-c-menu-toggle--m-split-button--m-action--m-primary--child--BackgroundColor
--pf-c-menu-toggle--m-split-button--m-action--m-primary--child--BorderLeftColor
--pf-c-menu-toggle--m-split-button--m-action--m-primary--child--active--BackgroundColor
--pf-c-menu-toggle--m-split-button--m-action--m-primary--child--focus--BackgroundColor
--pf-c-menu-toggle--m-split-button--m-action--m-primary--child--hover--BackgroundColor
--pf-c-menu-toggle--m-split-button--m-action--m-primary--m-expanded--child--BackgroundColor
--pf-c-menu-toggle--m-split-button--m-action--m-secondary--child--BorderLeftColor
--pf-c-menu-toggle--m-typeahead--c-text-input-group__utilities--c-button--PaddingRight
--pf-c-menu__breadcrumb--c-breadcrumb__heading--FontSize
--pf-c-menu__breadcrumb--c-breadcrumb__item--FontSize
--pf-c-menu__header--c-menu__item--BackgroundColor
--pf-c-menu__header--c-menu__item--MarginBottom
--pf-c-menu__header--c-menu__item--MarginLeft
--pf-c-menu__header--c-menu__item--MarginRight
--pf-c-menu__header--c-menu__item--MarginTop
--pf-c-menu__header--c-menu__item--PaddingBottom
--pf-c-menu__header--c-menu__item--PaddingLeft
--pf-c-menu__header--c-menu__item--PaddingRight
--pf-c-menu__header--c-menu__item--PaddingTop
--pf-c-menu__header--c-menu__item--focus--BackgroundColor
--pf-c-menu__header--c-menu__item--hover--BackgroundColor
--pf-c-menu__item-text--item-toggle-icon--MarginLeft
--pf-c-menu__item-toggle-icon--item-text--MarginLeft
--pf-c-menu__list--c-divider--MarginBottom
--pf-c-menu__list--c-divider--MarginTop
--pf-c-modal-box--c-button--Right
--pf-c-modal-box--c-button--Top
--pf-c-modal-box--c-button--sibling--MarginRight
--pf-c-modal-box--c-wizard--FlexBasis
--pf-c-modal-box__footer--c-button--MarginRight
--pf-c-modal-box__footer--c-button--MarginRight
--pf-c-modal-box__footer--c-button--sm--MarginRight
--pf-c-nav--c-divider--BackgroundColor
--pf-c-nav--c-divider--BackgroundColor
--pf-c-nav--c-divider--MarginBottom
--pf-c-nav--c-divider--MarginBottom
--pf-c-nav--c-divider--MarginTop
--pf-c-nav--c-divider--PaddingLeft
--pf-c-nav--c-divider--PaddingLeft
--pf-c-nav--c-divider--PaddingRight
--pf-c-nav--c-divider--PaddingRight
--pf-c-nav--c-menu--BackgroundColor
--pf-c-nav--c-menu--m-flyout--BoxShadow
--pf-c-nav--c-menu--m-flyout--Top
--pf-c-nav--c-menu--m-flyout--c-menu--Bottom
--pf-c-nav--c-menu--m-flyout--c-menu--Bottom
--pf-c-nav--c-menu--m-flyout--c-menu--Bottom
--pf-c-nav--c-menu--m-flyout--c-menu--Left
--pf-c-nav--c-menu--m-flyout--c-menu--Left
--pf-c-nav--c-menu--m-flyout--c-menu--Right
--pf-c-nav--c-menu--m-flyout--c-menu--Right
--pf-c-nav--c-menu--m-flyout--c-menu--Right
--pf-c-nav--c-menu--m-flyout--c-menu--Top
--pf-c-nav--c-menu--m-flyout--c-menu--Top
--pf-c-nav--c-menu--m-flyout--left-offset
--pf-c-nav--c-menu--m-flyout--m-left--Right
--pf-c-nav--c-menu--m-flyout--m-left--right-offset
--pf-c-nav--c-menu--m-flyout--m-top--Bottom
--pf-c-nav--c-menu--m-flyout__item--PaddingBottom
--pf-c-nav--c-menu--m-flyout__item--PaddingLeft
--pf-c-nav--c-menu--m-flyout__item--PaddingRight
--pf-c-nav--c-menu--m-flyout__item--PaddingTop
--pf-c-nav--c-menu--m-flyout__item--hover--after--BorderLeftColor
--pf-c-nav--c-menu--m-flyout__item--hover--after--BorderLeftWidth
--pf-c-nav--c-menu__item--FontSize
--pf-c-nav--c-menu__item--OutlineOffset
--pf-c-nav--c-menu__item--PaddingBottom
--pf-c-nav--c-menu__item--PaddingLeft
--pf-c-nav--c-menu__item--PaddingLeft
--pf-c-nav--c-menu__item--PaddingRight
--pf-c-nav--c-menu__item--PaddingRight
--pf-c-nav--c-menu__item--PaddingTop
--pf-c-nav--c-menu__item--after--BorderLeftColor
--pf-c-nav--c-menu__item--after--BorderLeftColor
--pf-c-nav--c-menu__item--after--BorderLeftColor
--pf-c-nav--c-menu__item--after--BorderLeftWidth
--pf-c-nav--c-menu__item--after--BorderLeftWidth
--pf-c-nav--c-menu__item--after--BorderLeftWidth
--pf-c-nav--c-menu__item--before--BorderBottomColor
--pf-c-nav--c-menu__item--before--BorderBottomWidth
--pf-c-nav--c-menu__item--before--BorderBottomWidth
--pf-c-nav--c-menu__item--m-current--BackgroundColor
--pf-c-nav--c-menu__item--m-current--after--BorderColor
--pf-c-nav--c-menu__item--m-current--after--BorderLeftWidth
--pf-c-nav--c-menu__item--xl--PaddingLeft
--pf-c-nav--c-menu__item--xl--PaddingRight
--pf-c-nav--c-menu__item-description--Color
--pf-c-nav--c-menu__list--PaddingBottom
--pf-c-nav--c-menu__list--PaddingTop
--pf-c-nav--c-menu__list-item--Color
--pf-c-nav--c-menu__list-item--active--Color
--pf-c-nav--c-menu__list-item--focus-within--BackgroundColor
--pf-c-nav--c-menu__list-item--focus-within--Color
--pf-c-nav--c-menu__list-item--hover--BackgroundColor
--pf-c-nav--c-menu__list-item--hover--Color
--pf-c-nav--c-menu__list-item--m-drill-up--before--BorderBottomWidth
--pf-c-nav--c-menu__list-item--m-drill-up__item--FontWeight
--pf-c-nav--m-light--c-divider--BackgroundColor
--pf-c-nav__item--item__link--after--Top
--pf-c-nav__item--item__link--after--Top
--pf-c-nav__item--item__link--after--Top
--pf-c-nav__item--m-flyout--c-menu--BackgroundColor
--pf-c-nav__item--m-flyout--c-menu--BoxShadow
--pf-c-nav__item--m-flyout--c-menu--Left
--pf-c-nav__item--m-flyout--c-menu--Top
--pf-c-nav__item--m-flyout--c-menu--c-menu--Top
--pf-c-nav__item--m-flyout--c-menu--c-menu--Top
--pf-c-nav__item--m-flyout--c-menu--left-offset
--pf-c-nav__item--m-flyout--c-menu--m-left--Right
--pf-c-nav__item--m-flyout--c-menu--m-left--right-offset
--pf-c-nav__item--m-flyout--c-menu--m-top--Bottom
--pf-c-nav__item--m-flyout--c-menu--m-top--bottom-offset
--pf-c-nav__item--m-flyout--c-menu--top-offset
--pf-c-nav__item--m-flyout--c-menu__item--Color
--pf-c-nav__item--m-flyout--c-menu__item--FontSize
--pf-c-nav__item--m-flyout--c-menu__item--OutlineOffset
--pf-c-nav__item--m-flyout--c-menu__item--PaddingBottom
--pf-c-nav__item--m-flyout--c-menu__item--PaddingLeft
--pf-c-nav__item--m-flyout--c-menu__item--PaddingRight
--pf-c-nav__item--m-flyout--c-menu__item--PaddingTop
--pf-c-nav__item--m-flyout--c-menu__item--before--BorderBottomColor
--pf-c-nav__item--m-flyout--c-menu__item--before--BorderBottomWidth
--pf-c-nav__item--m-flyout--c-menu__item--hover--after--BorderLeftColor
--pf-c-nav__item--m-flyout--c-menu__item--hover--after--BorderLeftWidth
--pf-c-nav__item--m-flyout--c-menu__item--hover--after--Top
--pf-c-nav__item--m-flyout--c-menu__item--hover--after--Top
--pf-c-nav__item--m-flyout--c-menu__item-description--Color
--pf-c-nav__item--m-flyout--c-menu__list-item--active--BackgroundColor
--pf-c-nav__item--m-flyout--c-menu__list-item--first-child--c-menu--Top
--pf-c-nav__item--m-flyout--c-menu__list-item--first-child__item--hover--after--Top
--pf-c-nav__item--m-flyout--c-menu__list-item--focus-within--BackgroundColor
--pf-c-nav__item--m-flyout--c-menu__list-item--hover--BackgroundColor
--pf-c-nav__subnav--c-divider--PaddingLeft
--pf-c-nav__subnav--c-divider--PaddingRight
--pf-c-number-input--c-form-control--Width
--pf-c-number-input--c-form-control--width-base
--pf-c-number-input--c-form-control--width-chars
--pf-c-number-input--c-form-control--width-icon
--pf-c-number-input--c-form-control--width-icon
--pf-c-number-input--m-status--c-form-control--width-icon
--pf-c-number-input__unit--c-input-group--MarginLeft
--pf-c-options-menu--c-divider--MarginBottom
--pf-c-options-menu--c-divider--MarginBottom
--pf-c-options-menu--c-divider--MarginTop
--pf-c-overflow-menu--c-divider--m-vertical--spacer
--pf-c-page__header-brand-link--c-brand--MaxHeight
--pf-c-page__header-tools--c-avatar--MarginLeft
--pf-c-page__header-tools--c-button--m-selected--before--BackgroundColor
--pf-c-page__header-tools--c-button--m-selected--before--BackgroundColor
--pf-c-page__header-tools--c-button--m-selected--before--BorderRadius
--pf-c-page__header-tools--c-button--m-selected--before--Height
--pf-c-page__header-tools--c-button--m-selected--before--Width
--pf-c-page__header-tools--c-button--m-selected--c-notification-badge--m-unread--after--BorderColor
--pf-c-page__header-tools--c-button--m-selected--notification-badge--m-attention--after--BackgroundColor
--pf-c-page__header-tools--c-button--m-selected--notification-badge--m-unread--after--BackgroundColor
--pf-c-page__header-tools--c-button--notification-badge--m-attention--after--BackgroundColor
--pf-c-page__header-tools--c-button--notification-badge--m-unread--after--BackgroundColor
--pf-c-page__header-tools-item--c-notification-badge--hover--BackgroundColor
--pf-c-page__sidebar-body--m-menu--c-context-selector--BorderBottomColor
--pf-c-pagination--c-options-menu--Display
--pf-c-pagination--c-options-menu--Display
--pf-c-pagination--c-options-menu--Display
--pf-c-pagination--c-options-menu--Display
--pf-c-pagination--c-options-menu--Visibility
--pf-c-pagination--c-options-menu--Visibility
--pf-c-pagination--c-options-menu--Visibility
--pf-c-pagination--c-options-menu--Visibility
--pf-c-pagination--c-options-menu__toggle--FontSize
--pf-c-pagination--child--MarginRight
--pf-c-pagination--child--MarginRight
--pf-c-pagination--child--MarginRight
--pf-c-pagination--m-bottom--child--MarginRight
--pf-c-pagination--m-bottom--child--MarginRight
--pf-c-pagination--m-bottom--child--md--MarginRight
--pf-c-pagination--m-bottom__nav-control--c-button--OutlineOffset
--pf-c-pagination--m-bottom__nav-control--c-button--OutlineOffset
--pf-c-pagination--m-bottom__nav-control--c-button--PaddingBottom
--pf-c-pagination--m-bottom__nav-control--c-button--PaddingBottom
--pf-c-pagination--m-bottom__nav-control--c-button--PaddingLeft
--pf-c-pagination--m-bottom__nav-control--c-button--PaddingRight
--pf-c-pagination--m-bottom__nav-control--c-button--PaddingRight
--pf-c-pagination--m-bottom__nav-control--c-button--PaddingTop
--pf-c-pagination--m-bottom__nav-control--c-button--PaddingTop
--pf-c-pagination--m-bottom__nav-control--c-button--md--PaddingBottom
--pf-c-pagination--m-bottom__nav-control--c-button--md--PaddingLeft
--pf-c-pagination--m-bottom__nav-control--c-button--md--PaddingRight
--pf-c-pagination--m-bottom__nav-control--c-button--md--PaddingTop
--pf-c-pagination--m-compact--child--MarginRight
--pf-c-pagination--m-display-full--c-options-menu--Display
--pf-c-pagination--m-display-full--c-options-menu--Visibility
--pf-c-pagination--m-display-summary--c-options-menu--Display
--pf-c-pagination--m-display-summary--c-options-menu--Visibility
--pf-c-pagination__nav-control--c-button--FontSize
--pf-c-pagination__nav-control--c-button--PaddingLeft
--pf-c-pagination__nav-control--c-button--PaddingLeft
--pf-c-pagination__nav-control--c-button--PaddingRight
--pf-c-pagination__nav-control--c-button--PaddingRight
--pf-c-pagination__nav-page-select--c-form-control--Width
--pf-c-pagination__nav-page-select--c-form-control--width-base
--pf-c-pagination__nav-page-select--c-form-control--width-chars
--pf-c-pagination__nav-page-select--child--MarginRight
--pf-c-popover--c-button--MarginLeft
--pf-c-popover--c-button--Right
--pf-c-popover--c-button--Top
--pf-c-popover--c-button--sibling--PaddingRight
--pf-c-popover--c-title--MarginBottom
--pf-c-search-input__utilities--c-button--PaddingLeft
--pf-c-search-input__utilities--c-button--PaddingRight
--pf-c-search-input__utilities--child--MarginLeft
--pf-c-select-menu--c-divider--MarginBottom
--pf-c-select-menu--c-divider--MarginBottom
--pf-c-select-menu--c-divider--MarginTop
--pf-c-select-menu--c-divider--MarginTop
--pf-c-select__toggle-wrapper--c-chip-group--MarginBottom
--pf-c-select__toggle-wrapper--c-chip-group--MarginTop
--pf-c-sidebar__main--child--MarginLeft
--pf-c-sidebar__main--child--MarginLeft
--pf-c-sidebar__main--child--MarginLeft
--pf-c-sidebar__main--child--MarginLeft
--pf-c-sidebar__main--child--MarginLeft
--pf-c-sidebar__main--child--MarginTop
--pf-c-sidebar__main--child--MarginTop
--pf-c-sidebar__main--child--MarginTop
--pf-c-sidebar__main--child--MarginTop
--pf-c-sidebar__main--child--MarginTop
--pf-c-slider__value--c-form-control--Width
--pf-c-slider__value--c-form-control--width-base
--pf-c-slider__value--c-form-control--width-chars
--pf-c-table--m-compact__toggle--c-button--responsive--MarginBottom
--pf-c-table--m-tree-view-grid--c-dropdown--MarginBottom
--pf-c-table--m-tree-view-grid--c-dropdown--MarginTop
--pf-c-table--m-tree-view-grid--c-table__action--PaddingBottom
--pf-c-table--m-tree-view-grid--c-table__action--PaddingLeft
--pf-c-table--m-tree-view-grid--c-table__action--PaddingTop
--pf-c-table--m-tree-view-grid--td--not--c-table__tree-view-title-cell--PaddingBottom
--pf-c-table--m-tree-view-grid--td--not--c-table__tree-view-title-cell--PaddingBottom
--pf-c-table--m-tree-view-grid--td--not--c-table__tree-view-title-cell--PaddingTop
--pf-c-table--m-tree-view-grid--td--not--c-table__tree-view-title-cell--PaddingTop
--pf-c-table--m-tree-view-mobile__tree-view-main--c-table__check--MarginRight
--pf-c-table--m-tree-view-mobile__tree-view-main--c-table__check--Order
--pf-c-table__column-help--c-button--MarginBottom
--pf-c-table__column-help--c-button--MarginTop
--pf-c-table__column-help--c-button--PaddingLeft
--pf-c-table__column-help--c-button--PaddingRight
--pf-c-table__draggable--c-button--MarginBottom
--pf-c-table__draggable--c-button--MarginLeft
--pf-c-table__draggable--c-button--MarginRight
--pf-c-table__draggable--c-button--MarginTop
--pf-c-table__favorite--c-button--Color
--pf-c-table__favorite--c-button--FontSize
--pf-c-table__favorite--c-button--MarginBottom
--pf-c-table__favorite--c-button--MarginBottom
--pf-c-table__favorite--c-button--MarginLeft
--pf-c-table__favorite--c-button--MarginLeft
--pf-c-table__favorite--c-button--MarginRight
--pf-c-table__favorite--c-button--MarginRight
--pf-c-table__favorite--c-button--MarginTop
--pf-c-table__favorite--c-button--MarginTop
--pf-c-table__favorite--m-favorited--c-button--Color
--pf-c-table__toggle--c-button--MarginBottom
--pf-c-table__toggle--c-button--MarginTop
--pf-c-table__toggle--c-button--m-expanded__toggle-icon--Rotate
--pf-c-table__toggle--c-button__toggle-icon--Rotate
--pf-c-table__toggle--c-button__toggle-icon--Transition
--pf-c-table__tree-view-main--c-table__check--MarginRight
--pf-c-table__tree-view-main--c-table__check--PaddingLeft
--pf-c-table__tree-view-main--c-table__check--PaddingRight
--pf-c-tabs--m-box__item-close--c-button--disabled--BackgroundColor
--pf-c-tabs--m-secondary__add--c-button--FontSize
--pf-c-tabs--m-secondary__item-close--c-button--FontSize
--pf-c-tabs__add--c-button--FontSize
--pf-c-tabs__add--c-button--FontSize
--pf-c-tabs__add--c-button--OutlineOffset
--pf-c-tabs__add--c-button--PaddingBottom
--pf-c-tabs__add--c-button--PaddingTop
--pf-c-tabs__item-close--c-button--FontSize
--pf-c-tabs__item-close--c-button--FontSize
--pf-c-tabs__item-close--c-button--OutlineOffset
--pf-c-tabs__item-close--c-button--PaddingBottom
--pf-c-tabs__item-close--c-button--PaddingLeft
--pf-c-tabs__item-close--c-button--PaddingTop
--pf-c-tabs__link--child--MarginRight
--pf-c-tabs__link--child--MarginRight
--pf-c-text-input-group__utilities--c-button--PaddingLeft
--pf-c-text-input-group__utilities--c-button--PaddingRight
--pf-c-text-input-group__utilities--c-button--PaddingRight
--pf-c-text-input-group__utilities--child--MarginLeft
--pf-c-toggle-group__item--item--MarginLeft
--pf-c-toolbar--c-divider--m-vertical--spacer
--pf-c-tree-view__node-count--c-badge--m-read--BackgroundColor
--pf-c-wizard__footer--child--MarginBottom
--pf-c-wizard__footer--child--MarginRight

```

Reply:
Sure, I'll open an issue; I think you made your point.